src/test/kotlin/oop/Observer/subscription/ObserverWithSubscription.kt

```diff
@@ -0,0 +1,97 @@
+package oop.Observer.subscription
+
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.matchers.shouldEqual
+import io.kotlintest.specs.WordSpec
+
+class ObserverWithSubscription : WordSpec() {
+
+  init {
+    "SubjectPublish" should {
+      val observers = mutableListOf<Observer<String>>()
+      val subject: PublishSubject<String> = PublishSubject(observers)
+
+      "register a observer" {
```

Comment:
Yes I was starting with a JUnit test, but how I have created a one file for 3 classes, I prefer use this kind of test to separate differents things, like Subject and Observer 👍